commands/user_e2e_test.go

```diff
@@ -136,3 +138,77 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestResetUserMfaCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	user, appErr := s.th.App.CreateUser(&model.User{Email: s.th.GenerateTestEmail(), Username: model.NewId(), Password: model.NewId(), MfaActive: true, MfaSecret: "secret"})
+	s.Require().Nil(appErr)
+
+	s.Run("Reset user mfa", func() {
+		printer.Clean()
+
+		previousVal := s.th.App.Config().ServiceSettings.EnableMultifactorAuthentication
+		s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableMultifactorAuthentication = true })
+		defer func() {
+			s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableMultifactorAuthentication = *previousVal })
+		}()
+
+		s.th.AddPermissionToRole(model.PERMISSION_EDIT_OTHER_USERS.Id, model.SYSTEM_ADMIN_ROLE_ID)
+		defer func() {
+			s.th.RemovePermissionFromRole(model.PERMISSION_EDIT_OTHER_USERS.Id, model.SYSTEM_ADMIN_ROLE_ID)
+		}()
```

Comment:
I don't think `SYTEM_ADMIN` needs an extra permission to do things :)
```suggestion
```

Reply:
Hmm, I see, seems my bad on that one, still learning about flow in mattermost,
It seems to work well with local and system_admin, let me adjust to that 

thanks @isacikgoz :+1: